jwst/skymatch/skyimage.py

```diff
@@ -1,4 +1,4 @@
-"""
+spherical_geometry"""
```

Comment:
@stsci-hack Undescore begins "italics" but it never closes it. You should use double back-ticks for package name.

Reply:
@stsci-hack Actually, disregard my previous comment - simply remove `spherical_geometry` before the docstring. What is it doing there?

Reply:
This change is fine...